pytube/extract.py

```diff
@@ -67,10 +64,8 @@ def is_private(watch_html):
         "\"simpleText\":\"Private video\"",
         "This video is private."
     ]
-    for string in private_strings:
-        if string in watch_html:
-            return True
-    return False
+    
```

Comment:
Just nitpicking, but  it's best to keep the same consistent line spacing style, so I would remove that empty line.

Reply:
ight